Makefile

```diff
@@ -262,6 +262,14 @@ unit-tests-race: buildkitd
 		--opt build-arg:TESTPKGS=$(TESTPKGS) \
 		$(COMMON_ARGS)
 
+.PHONY: integration.test
+integration.test: buildkitd
+	@$(BINDIR)/buildctl --addr $(BUILDKIT_HOST) \
+		build \
+		--output type=local,dest=build \
```

Comment:
If this is output to `build`, then it will make it into our release. I think we should add this too `./bin` (its ignored by git).

Reply:
thanks for the hint!

Reply:
in the long run, I feel our integration test might be part of the release (to make sure we have some level of backward or forward compatibility)

Reply:
Yeah good idea. In that case should we leave it?